Client/Pages/GetFileTest.razor.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Client.ViewModels;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+using Client.Services;
+
+namespace Client.Pages
+{
+    public partial class GetFileTest : ComponentBase
+    {
+        public static FileIDInputVM FileObj { get; set; } = new FileIDInputVM();
+        public async void OnValidSubmit(EditContext Context)
+        {
+            HttpService wc = new HttpService();
+            await HttpService.PostAsync("api/FileInterface/Download", FileObj.FileGUID);
```

Comment:
This is not calling the wc objects PostAsync method

Reply:
Good call, I missed that.